docs-src/modules/getting-started/pages/hello-world-cli.adoc

```diff
@@ -234,94 +226,101 @@ This makes it easier to see how a single entity changes over time.
 Purchase | when(Purchase.customer_id == "patrick")
 ----
 
-In this example, we build a pipeline of functions using the `|` character.
-We begin with the timeline produced by the table `Purchase`, then filter it to the set of times where the purchase's customer is `"patrick"` using the `xref:fenl:catalog.adoc#when[when()]` function.
-Execute the query using the CLI as before.
-
 [source,bash]
 ----
-cat query.fenl | kaskada-cli query run --stdout --response-as csv
+cat query.fenl | ./kaskada-cli query run --stdout --response-as csv
 ----
+In this example, we build a pipeline of functions using the `|` character.
+We begin with the timeline produced by the table `Purchase`, then filter it to the set of times where the purchase's customer is `"patrick"` using the `xref:fenl:catalog.adoc#when[when()]` function.
 
-As you begin to better understand your data you can start using
-aggregations over your data such as the `max()` function:
+Kaskada's query language provides a rich set of operations for reasoning about time.
```

Comment:
maybe link to the catalogue here, same style as "go here to find out more" that we have throughout the docs